docs/pages/database-access/guides/elastic.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Database Access with Elasticsearch
+description: How to configure Teleport Database Access with Elasticsearch.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.3"
+>
+  Database access for Elasticsearch is available starting from Teleport `10.3`.
+</Details>
+
+This guide will help you to configure secured access to an Elasticsearch database using Teleport Dabatase Access.
+
+## Prerequisites
+
+- A self-hosted Elasticsearch database. Elastic cloud [does not support client certificates](https://www.elastic.co/guide/en/cloud/current/ec-restrictions.html#ec-restrictions-security).
+
+- A host where you will run the Teleport Database Service. Teleport version 10.3
+  or newer must be installed. <ScopedBlock scope={["oss", "enterprise"]}> This can also be the same instance of Teleport running your Auth and/or Proxy service(s).</ScopedBlock>
```

Comment:
I think we can remove the "Teleport version 10.3
  or newer must be installed" note and the Installation link since we include instructions for installing Teleport on the Database Service host later.

Reply:
@ptgott what if a reader already has a Teleport instance running the Database Access service for other databases and wants to add Elasticsearch to it, and is using a version >10.3?

Reply:
I think we can make this more explicit, maybe something like:

```markdown
- A host where you will install and run the Teleport Database Service. 
  If you are already running the Teleport Database Service, you must 
  ensure that it uses Teleport version 10.3 or newer in order to connect to
  Elasticsearch. 
```